docs/freqai.md

```diff
@@ -408,6 +408,11 @@ It is common to want constant retraining, in whichcase, user should set `live_re
 
 ### Controlling the model learning process
 
+Depending on what AI model to be used, these parameter names could be different. For example, the accepted parameters for the `Catboost`
+models are `data_split_parameters`, `n_estimators` and etc. For the model like SVM regression model, the accepted parameters are different.
```

Comment:
This is untrue, `data_split_parameters` is not unique to Catboost. `data_split_parameters` are used for all FreqAI models which require a data split. Do you mean `model_training_parameters`? The dict entry for `model_training_parameters` in the user config is common for all models, but the contents of that particular dict entry depends on the type of model being employed (catboost vs lightgbm vs tensorflow).

Reply:
Yes, you are right. I put wrongly the `data_split_parameters` which is quite common for the /val/test dataset splits. I wanted to say the training parameters could be different for one ML approach from another. So, feel free to add the correspond training parameters

Reply:
This PR makes me think it could be useful to split out some sections for various models. “Common catboost regressor considerations” “common lightgbm considerations” “common tensorflow considerations” etc. Where we discuss some freqai related interactions which each library.